modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/rebalance/DistributionZoneRebalanceEngineTest.java

```diff
@@ -545,4 +557,12 @@ private void watchListenerOnUpdate(int zoneId, Set<String> nodes, long rev) {
 
         watchListener.onUpdate(evt);
     }
+
+    private int getZoneId(String zoneName) {
```

Comment:
I see that you have this method copy-pasted several times, lets move it to `org.apache.ignite.internal.distributionzones.DistributionZonesTestUtil`

Reply:
fix it